agent/cita_monitor_agent.py

```diff
@@ -303,6 +303,8 @@ def exporter():
     else:
         print(genesis_block_info)
     block_number_info = class_result.block_number()
+    hex_number = 0x1
+    previous_hex_number = 0x1
```

Comment:
`previous_hex_number` means `hex_number-1`, so here it should be `0x0`.

Reply:
@flyq can you update this？

Reply:
already update it

Reply:
please use amend to combine your commits, see https://help.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message

Reply:
can you amend your commits?

Reply:
I don't know how to amend the commits, so I think it is a better way that closing this pr and you push a commit to fix this. @rainchen

Reply:
@flyq try these steps:

1. git checkout -b "fix-agent-hex-humber-init-issue"
2. git reset --soft HEAD^
3. git reset --soft HEAD^
4. git add agent/cita_monitor_agent.py
5. git commit -am "fix cita_monitor_agent.py for hex_number initialization issue"
6. git push origin fix-agent-hex-humber-init-issue

Reply:
@flyq would you please handle this? we're going to release a new version, will be great to have this issue fixed

Reply:
@rainchen thx for your steps, I follow your advice: https://github.com/citahub/cita-monitor/pull/162

PS: I will learn more about git